README.md

```diff
@@ -138,10 +138,10 @@ contracts – best practices to adopt](https://www.lfenergy.org/wp-content/uploa
 ## Funding
 
 * [Nadya Eghbal's "Lemonade Stand"](https://github.com/nayafia/lemonade-stand) - A comprehensive list of funding methods for open source projects
-* [Open Source Software Funding Platforms Registry](http://oss.fund/) - List of platforms for funding open source software development
+* [Open Source Software Funding Platforms Registry](https://www.oss.fund/) - List of platforms for funding open source software development
 * [Open Collective](https://opencollective.com/) - Open source platform for managing and connecting projects, fiscal sponsors, and financial contributors
 * [Decentralized funding? An analysis of three programs](https://nadiaeghbal.com/grant-programs) - Analysis of some programs to fund open source development in an open and decentralized way through development grants
-* [COSSI: Commercial Open Source Software Company Index](http://oss.cash/) - List of companies which make more than 100 million USD revenue from open source software
```

Comment:
The list still includes only companies which make at least 100 million USD in revenue, doesn't it?

Reply:
I think so, although it might be older in the data that it shows. I captured the link from https://jjacks.svbtle.com/commercial-open-source-paradox

https://www.coss.community/jj/the-100-billion-commercial-oss-index-cossi-511h - written by the same person, captures a screenshot of the now defunct url and it looks surprisingly similar. that's how I ended up with that new link

Reply:
I added these references to the git commit message for future reference.